driver/web/docs/schemas/application/Survey.yaml

```diff
@@ -0,0 +1,25 @@
+type: object
+properties:
+  id:
+    readOnly: true
+    type: string
+  user_id:
+    type: string
+  creator_id:
+    type: string
+  org_id:
+    type: string
+  app_id:
+    type: string
+  stats:
+    type: object
+  questions:
+    type: object
```

Comment:
We should set `additionalProperties` on `questions` as follows to make it clear what the type of the values are in the docs:

```yaml
additionalProperties: 
      $ref: "./SurveyData.yaml"
```